MAUI/ImageEditor/toolbar-customization.md

```diff
@@ -0,0 +1,541 @@
+---
+layout: post
+title: Toolbar Customization in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about toolbar customization support in the Syncfusion .NET MAUI ImageEditor (SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Toolbar Customization in .NET MAUI Image Editor (SfImageEditor)
+
+You can customize the color palette, visibility, and appearance of each toolbar item.
+
```

Comment:
Add topics in below order

1. Toolbar style customization
 - Mention about ToolbarSettings, Toolbar background, icon color, disabled color, icon size
 2. Toolbar orientation
 3. Toolbar Position
 4. Toolbar size customization
 5. Customize default crop option => ToolbarSettings.CropTypes
 6. Customize default Effects option => ToolbarSettings.Effects
 7. Customize default shape option => ToolbarSettings.Shapes
 8. Customize default color palette colors => ToolbarSettings.Colors
 - mention this is common for Text, Shape and Pen
 9. Default Toolbar items => Mention all items added in default toolbar
 10. How to get an item from inbuilt top toolbar
 11. How to get an item from inbuilt  bottom toolbar
 12. How to get an item from sub toolbar
 13. How to change item visibility
 14. How to change item enable state
 15. How to change item view
 16. How to add a new sub toolbar by clearing existing sub toolbar => add for crop
 17. Add an item with the existing toolbar.
 18. Add new custom toolbar = auto generate false
 19. ToolbareItem selected event